label_sync/labels.yaml

```diff
@@ -8,6 +8,12 @@
 ---
 default:
   labels:
+    - color: e11d21
+      description: Categorizes an issue or PR as actively needing an API review. See https://git.k8s.io/community/sig-architecture/api-review-process.md
+      name: api-review
```

Comment:
kind/api-review? 

Ideally we try and fit this inside one of the other top-level categories. otherwise lgtm

Reply:
the label is a signal the PR is ready for API review (has met prereqs, is not WIP, etc)

Reply:
I considered`needs-api-review`, but decided against it because it's intended to be a human-added label, unlike the existing `needs-...` labels

Reply:
This is required in every repo in every org? Or is this just k/k (and maybe a couple other repos) thing?

Reply:
In general, it applies as wide as the kind/api-change label